source/examples/functions/MceStartStop/definition.yaml

```diff
@@ -1,6 +1,11 @@
 author: Rioual
 comment: Start/stop logic + speed override
 changelog:
+  - version: 0.7.0
+    date: 2024-12-13
+    author: Rioual
+    changed:
+      - wait for system running when resuming a trajectory
```

Comment:
How about "add check for system state to prevent flipping between states"?

Reply:
Done